### ISSUE TITLE
zdtm/fdt_shared: stack smashing detected

### ISSUE THREAD

**Issue by avagin** [xemul/criu#388]
```
26378 futex(0x55a8305141b0, FUTEX_WAKE, 2147483647) = 0
26378 rt_sigreturn({mask=[]})           = 202
26378 futex(0x55a8305141b0, FUTEX_WAIT, 0, NULL) = -1 EAGAIN (Resource temporarily unavailable)
26378 read(128, "Hello World!\0", 13)   = 13
26378 open("/dev/tty", O_RDWR|O_NOCTTY|O_NONBLOCK) = -1 ENXIO (No such device or address)
26378 writev(2, [{"*** ", 4}, {"stack smashing detected", 23}, {" ***: ", 6}, {"./fdt_shared", 12}, {" terminated\n", 12}], 5) = 57
26378 mmap(NULL, 4096, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_ANONYMOUS, -1, 0) = 0x7f4195a65000
26378 rt_sigprocmask(SIG_UNBLOCK, [ABRT], NULL, 8) = 0
26378 rt_sigprocmask(SIG_BLOCK, ~[RTMIN RT_1], [], 8) = 0
26378 getpid()                          = 50
26378 gettid()                          = 50
26378 tgkill(50, 50, SIGABRT)           = 0
26378 rt_sigprocmask(SIG_SETMASK, [], NULL, 8) = 0
26378 --- SIGABRT {si_signo=SIGABRT, si_code=SI_TKILL, si_pid=50, si_uid=18943} ---
```
Reported-by: @rodrigo-bruno

**Comment by avagin** [comment]
@rodrigo-bruno Could you check that you have this commit:
commit a01d879b627f72eddd8cd3201da83c5182c485a1
Author: Michael Holzheu <holzheu@linux.vnet.ibm.com>
Date:   Fri Jun 30 20:31:49 2017 +0200

    zdtm/static/fdt_shared: Fix buffer size

**Comment by avagin** [comment]
It was fixed by a01d879
